source/assets/sass/config/_scale.scss

```diff
@@ -0,0 +1,17 @@
+// Spacing Sizes
+// -------------
+/// @group scale
+
+$gutter: 1rem; // 16
+$gutter-quarter: calc(var(--gutter) * 0.25); // 4
+$gutter-half: calc(var(--gutter) * 0.5); // 8
+$gutter-minus: calc(var(--gutter) * 0.75); // 12
+$gutter-plus: calc(var(--gutter) * 1.25);
+$gutter-sesqui: calc(var(--gutter) * 1.5);
+$gutter-double: calc(var(--gutter) * 2);
+$gutter-triple: calc(var(--gutter) * 3);
```

Comment:
@mirisuzanne I went with the main word first (gutter) and then the modifier like "double" afterward since their styleguide explains why they go with that method.

Reply:
I like it, we should consider it for our other work. :)

Reply:
I am a big fan of it but you have to train your brain a little to not type `half-` first :)